### PR TITLE
Update changelog with recent merged PRs

### PR DESCRIPTION
**Pull request by bessey**
Title says it all.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
 ## [Unreleased]
-### Added
-...
 ### Changed
-...
+The default for the following options has changed:
+- parens_in_def: ~~dynamic~~ > yes
+- last_has_comma: ~~dynamic~~ > always
+### Removed
```
Comment (splattael) [inline CHANGELOG.md]: Add a newline.

**Comment by bessey** [comment]
Hehe, I face a difficult choice with 2 reviewers requesting opposing things! This is a tricky one for me, I am personally greatly appreciative of the contributions we've had recently, including from yourself @mjago, but I don't necessarily believe the changelog is the right place to document that appreciation.

Perhaps when we cut a release we can give thanks there?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<!--
+  Thank you for contributing to rufo!
+  Please remember to update the CHANGELOG with the contents of this PR.
+-->
```
Comment (bessey) [inline .github/PULL_REQUEST_TEMPLATE.md]: For those unfamiliar, the contents of this file show up in the description of any new PRs created through the GH web UI. As its am HTML comment even if its not deleted by the submitter, it doesn't appear in the description.

I've found this a useful way to communicate contribution guidelines in other projects, so I've added it here too.

Comment (gingermusketeer) [reply .github/PULL_REQUEST_TEMPLATE.md]: This is a great idea!

**Comment by mjago** [comment]
@splattael are you happy with us updating the changelog?

**Comment by mjago** [comment]
Thanks 😄

**Pull request merged**